.changeset/tricky-donkeys-do.md

```diff
@@ -0,0 +1,5 @@
+---
+'@techdocs/cli': minor
+---
+
+support passing additional mkdocs-server CLI parameters when run in containerized mode
```

Comment:
Can we either include the new parameters here or mention to run `techdocs-cli --help` for more details? This just helps Adopters learn a bit more about this change when they review the release (changesets get rollup into release notes)

Reply:
Yes, I've included details of new passable `mkdocs` params in latest commit. PTAL and thanks for the review!